test-fixtures/target_secret_postgres_test.go

```diff
@@ -0,0 +1,61 @@
+package dockertest
+
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/hashicorp/vault-benchmark/helper/dockertest"
+	"github.com/hashicorp/vault-benchmark/helper/dockertest/dockerjobs"
+)
+
+func TestPostgres_Secret_Docker(t *testing.T) {
+	t.Parallel()
+
+	// Create Vault Container
+	vaultCleanup, vaultContainerIPAddress := dockertest.CreateVaultContainer(t)
+	defer vaultCleanup()
+
+	// Create Postgres Container
+	postgresCleanup, postgresContainerIPAddress := dockertest.CreatePostgresContainer(t)
+	defer postgresCleanup()
+
+	cleanupHCL, newHCLFile := editHCL(t, "./configs/postgres.hcl", "<container_addr>", postgresContainerIPAddress)
```

Comment:
Looking at the sdk, I see its possible to set the hostname: https://github.com/hashicorp/vault/blob/main/sdk/helper/docker/testhelpers.go#L363.

Have you considered doing it this way so we can have a static value for the hostname?

Reply:
The postgres and benchmark conatiners are both on the default bridge network, so I believe the containers cannot communicate with eachother via hostname, but only by IP address. If they were on a user created network we could use the hostname. We had that logic in an earlier [PR](https://github.com/hashicorp/vault-benchmark/pull/123/commits/c08b8c84b792103b5bfd361d372ea1fa84066d70) but removed it because it seemed like overkill.